projects/Kaidan/default.nix

```diff
@@ -45,5 +45,12 @@
       };
     };
   };
-  nixos.modules.services.kaidan.module = null;
+
+  nixos.demo.vm = {
+    module = ./programs/demo.nix;
+    tests.demo.module = null;
+    description = ''
+      How to use Kaidan
+    '';
```

Comment:
I think this is supposed to be filled in with a helpful description for users, or a more clear "This is a placeholder, a proper description still needs to be added" kinda text? Preferably the former I think.